usecases/backup/backend.go

```diff
@@ -124,7 +126,8 @@ Loop:
 
```

Comment:
when using similar functions like WithLabelValues it is in my opinion better to have labels and values not only values.
something like ("backend", "s3", "class", "A") instead ("s3", "A")